app/include/user_config.h

```diff
@@ -136,6 +136,9 @@
 //#define TIMER_SUSPEND_ENABLE
 //#define PMSLEEP_ENABLE
 
+// The net module optionally offers net info functionnality. Uncomment the following
+// to enable the functionnality.
+//#define NET_INFO_ENABLE 
```

Comment:
```suggestion
#define NET_INFO_ENABLE 
```

I was under the impression that we had consensus, albeit a vague one, between @nwf, @TerryE  and myself that it should be enabled by default? If not then I'll adjust the cloud builder to enable it.

Reply:
If we are dropping this down to `net.ping()`then it makes sense to make this define `NET_PING_ENABLE`.

Reply:
If we are dropping this down to `net.ping()`then it makes sense to make this define `NET_PING_ENABLE`.

Reply:
If we are dropping this down to `net.ping()`then it makes sense to make this define `NET_PING_ENABLE`.

Reply:
If we are dropping this down to `net.ping()`then it makes sense to make this define `NET_PING_ENABLE`.